src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const searchMovie = (movie: Movie) => {
+    const queryToLowerCase = query.toLowerCase().trim();
+    const titleToLowerCase = movie.title.toLowerCase();
+    const descriptionToLowerCase = movie.description.toLowerCase();
```

Comment:
Let's go rename this variables: `loverCasedQuery`, `lowerCasedTitle`, because names of your variables sounds like functions